pandas/core/style.py

```diff
@@ -255,6 +272,75 @@ def _translate(self):
                     precision=precision, table_styles=table_styles,
                     caption=caption, table_attributes=self.table_attributes)
 
+    def format(self, formatter, subset=None):
+        '''
+        Format the text display value of cells.
+
+        .. versionadded:: 0.18.0
+
```

Comment:
as an aside, we might want to allow ``includes, excludes`` kws to automatically apply to certain dtypes (e.g. just pass thru to ``.select_dtypes``) as this can be an alternative to subset, though this might be stretching the api a bit. *maybe* we could overload ``subset`` a bit to handle this.